src/main/java/seedu/address/model/contact/Telegram.java

```diff
@@ -32,7 +32,7 @@ public Telegram(String id) {
      * Returns true if a given string is a valid phone number.
      */
     public static boolean isValidId(String test) {
-        return test.matches(VALIDATION_REGEX);
+        return test.matches(VALIDATION_REGEX) && !test.isEmpty();
```

Comment:
This violates SLAP. The validation regex can enforce the non-empty condition if you change the `*` at the end to a `+`